float32 velocity
float32 acceleration
float32 deceleration
float32 force
---
bool success
string message
